tests/phpunit/MediaWiki/TitleFactoryTest.php

```diff
@@ -56,6 +56,21 @@ public function testNewFromIDs() {
 		);
 	}
 
+	public function testNewFromIDsEmpty() {
+
+		$instance = new TitleFactory();
+		$input = [];
+
+		$out = $instance->newFromIDs($input);
+
+		$this->assertCount( 0, $out );
+
+		$this->assertInternalType(
+			'array',
+			$out
+		);
+	}
```

Comment:
Better than nothing, but definitly not testing everything.

Also not following the MW coding style

Reply:
I agree, added a test where all used services are mocked.
Ideally this would use `MediaWikiIntegrationTestCase` if/when it is available in SMW

Reply:
Why would `MediaWikiIntegrationTestCase` not be available in SMW?

Reply:
Totally missed the `Integration` tests folder.  
Moved the test accordingly